docs/source/new-project.rst

```diff
@@ -538,7 +536,7 @@ The :func:`~~pytorch_lightning.core.lightning.LightningModule.log` method has a
 - prog_bar (logs to the progress bar)
 - logger (logs to the logger like Tensorboard)
 
-Depending on where log is called from, Lightning auto-determines the correct mode for you. But of course
+Depending on where the log is called from, Lightning auto-determines the correct mode for you. But of course
```

Comment:
log is the name of the method, so that was correct, but now I believe this is wrong. 
Should be "... where log is called from..." or "...where the log method is called from..."